src/Facebook/Facebook.php

```diff
@@ -586,4 +589,72 @@ public function videoToUpload($pathToFile)
     {
         return new FacebookVideo($pathToFile);
     }
+
+    /**
+     * Upload video
+     *
+     * @param int $target
+     * @param string $pathToFile
+     * @param string $accessToken
+     * @param int $maxTransferTries
+     *
+     * @return array
+     *
+     * @throws FacebookSDKException
+     */
+    public function uploadVideo($target, $pathToFile, $accessToken, $maxTransferTries = 5)
+    {
+        $uploader = $this->getResumableUploader($accessToken, $maxTransferTries);
+        $endpoint = '/'.$target.'/videos';
```

Comment:
We shouldn't auto-suffix the endpoint with `videos` since it might not always be that and none of the other parts of the SDK auto-suffix endpoints. :)